frontend/javascripts/oxalis/view/right-border-tabs/skeleton_tab_view.tsx

```diff
@@ -737,11 +736,20 @@ class SkeletonTabView extends React.PureComponent<Props, State> {
         },
         {
           key: "handleNmlDownload",
-          onClick: this.handleNmlDownload,
-          title: "Download visible trees as NML",
+          onClick: () => this.handleNmlDownload(false),
           icon: <DownloadOutlined />,
           label: "Download Visible Trees",
+          title: "Download Visible Trees as NML",
         },
+        this.props.isSkeletonLayerTransformed
+          ? {
+              key: "handleNmlDownloadTransformed",
+              onClick: () => this.handleNmlDownload(true),
+              icon: <DownloadOutlined />,
+              label: "Download Visible Trees (Transformed)",
+              title: "The currently active transformation will be applied to each node.",
+            }
```

Comment:
I'm not sure whether I would offer the two download options. Instead, I'd always use the untransformed node positions and possibly include the active transforms if the skeleton is transformed. Consumers of the nml would need to apply the transformation themselves if they'd like to. If I remember correctly this is also what TrakEM does and the positions no longer would need to be transformed/untransformed during nml export/import.